paddle/capi/main.h

```diff
@@ -26,6 +26,11 @@ extern "C" {
  */
 PD_API paddle_error paddle_init(int argc, char** argv);
 
+/**
+ * Initialize the thread environment of Paddle.
+ */
+PD_API paddle_error paddle_init_thread();
```

Comment:
Since some previous CPU users did not use this interface. Here need to indicate only GPU need this interface.

Reply:
Done.